lib/configs/internal.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+module.exports = {
+    parser: 'babel-eslint',
+    extends: 'plugin:@hapi/recommended',
+    parserOptions: {
+        loc: true,
+        comment: true,
+        range: true,
```

Comment:
I don't believe that these are needed anymore, for two reasons:
 - These are esprima options, and lab doesn't use esprima anymore.  babel-eslint uses @babel/parser under the hood.
 - These options were only used to centralize the esprima configuration between linting and parsing when generating coverage reports.  There's some good context for that here: https://github.com/hapijs/lab/commit/f88a72b0a22c252ea88f67db1476a2258921b9ce
```suggestion
```

Reply:
Alright, thank you @devinivy . I didn't notice that `espree` was not used anymore. I'll commit your suggestion.